src/main/java/uk/gov/ch/developer/docs/session/SessionFactory.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.ch.developer.docs.session;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.session.Session;
+import uk.gov.companieshouse.session.store.Store;
+
+/**
+ * This is a wrapper component for {@link uk.gov.companieshouse.session.SessionFactory} that allows
+ * for mocking return results, rather than the implemented static methods.
+ *
+ * @see uk.gov.companieshouse.session.SessionFactory#getSessionByCookieId(String)
+ * @see uk.gov.companieshouse.session.SessionFactory#getSessionByCookieId(Store, String)
+ */
+@Component
+public class SessionFactory {
+
+    private SessionFactory() {
+    }
+
+    public Session getSessionByCookieId(final String cookieId) {
+        return uk.gov.companieshouse.session.SessionFactory.getSessionByCookieId(cookieId);
+    }
+
+    public Session createSession() {
+        return getSessionByCookieId(null);
```

Comment:
I agree we should leave this class, can we combine this with above method & change the javadoc.
```suggestion
        return uk.gov.companieshouse.session.SessionFactory.getSessionByCookieId(null);
```

Reply:
I would much rather it shadowed the original static class. I see your point, but if we mock out one, I would prefer to keep the same chain of calls.